### PR TITLE
Added FollowTle and Tracking instructions

### PR DESCRIPTION
**Pull request by NickHardy**
Hi Dale,
I have added some instructions.
- Follow TLE
- Set Tracking
Please let me know what you think.

### CODE REVIEW

File: Tracking/Tracking.cs [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+            } catch {
+                throw;
+            } finally {
+                progress?.Report(new ApplicationStatus() { Status = string.Empty });
+            }
+
+            return;
+        }
+
+        public IList<short> M3Ports => ItemLists.M3Ports;
```
Comment (daleghent) [inline Tracking/Tracking.cs:78]: This doesn't appear to be used anywhere

File: Tracking/Tracking.cs [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+using System.Diagnostics;
+using System.Globalization;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Web;
+
+namespace DaleGhent.NINA.PlaneWaveTools.Tracking {
+
+    [ExportMetadata("Name", "Set tracking")]
```
Comment (daleghent) [inline Tracking/Tracking.cs]: Capitalize the T - `Set Tracking`

File: Tracking/Tracking.cs [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+    [ExportMetadata("Description", "Start or stop tracking")]
+    [ExportMetadata("Icon", "SpeedometerSVG")]
+    [ExportMetadata("Category", "PlaneWave Tools")]
```
Comment (daleghent) [inline Tracking/Tracking.cs:38]: I wonder if this `Set Tracking` instruction should be placed into its own category with `TLE Follow` (or `Load TLE` in its alternate name) - perhaps something like `PlaneWave Tools - Satellites`

File: TLE/TLEControl.cs [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+using System.Diagnostics;
+using System.Globalization;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Web;
+
+namespace DaleGhent.NINA.PlaneWaveTools.TLE {
+
+    [ExportMetadata("Name", "TLE follow")]
```
Comment (daleghent) [inline TLE/TLEControl.cs:35]: Perhaps this should be named `Load TLE` since that's what this is doing with PWI4?

File: TLE/TLEControl.cs [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+    [ExportMetadata("Description", "Start following a satellite")]
+    [ExportMetadata("Icon", "PlatesolveAndRotateSVG")]
+    [ExportMetadata("Category", "PlaneWave Tools")]
```
Comment (daleghent) [inline TLE/TLEControl.cs:38]: See comment regarding Category name that I made in `Tracking.cs`

File: TLE/TLEControl.cs [hunk 6]
```diff
@@ -0,0 +1,229 @@
...
+                    double.TryParse(status["mount.axis0.dist_to_target_arcsec"], out double axis0Distance);
+                    double.TryParse(status["mount.axis1.dist_to_target_arcsec"], out double axis1Distance);
+                    Axis0Distance = axis0Distance;
+                    Axis1Distance = axis1Distance;
+                    progress?.Report(new ApplicationStatus() { Status = $"Distance to target: {axis0Distance}-{axis1Distance}" });
+                    await CoreUtil.Delay(500, ct);
+                    if (Math.Abs(axis0Distance) > 5 || Math.Abs(axis1Distance) > 5)
+                        _tracking = DateTime.Now;
+                    if (_timeout.AddMinutes(1) < DateTime.Now)
+                        throw new SequenceEntityFailedException("Timed out after 1 minute.");
```
Comment (daleghent) [inline TLE/TLEControl.cs]: Timed out doing what?

**Review by daleghent — CHANGES_REQUESTED** [review]

File: Tracking/Tracking.cs (hunk 1 above)
Comment (NickHardy) [reply Tracking/Tracking.cs:78]: Oops. I took it out now.

File: Tracking/Tracking.cs (hunk 2 above)
Comment (NickHardy) [reply Tracking/Tracking.cs]: Done

File: Tracking/Tracking.cs (hunk 3 above)
Comment (NickHardy) [reply Tracking/Tracking.cs:38]: Set Tracking is just like the normal Set Tracking instruction but only the on and off options.

File: TLE/TLEControl.cs (hunk 4 above)
Comment (NickHardy) [reply TLE/TLEControl.cs:35]: It starts following and tracking right away.

File: TLE/TLEControl.cs (hunk 5 above)
Comment (NickHardy) [reply TLE/TLEControl.cs:38]: Might not be needed

File: TLE/TLEControl.cs (hunk 6 above)
Comment (NickHardy) [reply TLE/TLEControl.cs]: Failed to follow the target within a minute. Throw an error and stop tracking to be on the safe side.
I did improve the message a bit.

**Comment by NickHardy** [comment]
Hi Dale,
I will test the normal Set Tracking instruction on a real scope to see if it's working or not.
Feel free to wait before merging.

**Review by daleghent — APPROVED** [review]

**Pull request merged**